apps/server/src/shared/repo/user/user.repo.ts

```diff
@@ -163,6 +163,10 @@ export class UserRepo extends BaseRepo<User> {
 		return promise;
 	}
 
+	async delete(user: User): Promise<void> {
+		await this._em.removeAndFlush(user);
+	}

```

Comment:
Please replace it with the method that allows to delete the whole document by the primary key only (which is in fact the userId) - thanks to this you'll not need to do the document fetch first in the `deleteUser()` method in the `UserService`.

Reply:
ok fixed this